tools/install.sh

```diff
@@ -336,6 +344,18 @@ elif [ "$LINUX_VERSION" == "BackBox" ] || \
 		easy_install -U distribute
 		apt-get install -y postgresql-server-dev-all &> /dev/null
 	fi
+elif [ "$LINUX_VERSION" == "Arch" ]; then
+	pacman --noconfirm -S freetype2 python python-pip pkg-config
```

Comment:
Is this installing Python2 or Python3, because 1.8 (the latest release) is the last one in which we support Python2.